### PR TITLE
`Tree.shinyInput` added

### PR DESCRIPTION
**Pull request by pawelchabros**
Issue #60 

Changes compared to regular `Tree` component:

* specifying `id` not needed
* don't need to write any handler functions
* tree state handled on client side
* selected nodes highlighted
* possibility to select multiple nodes (hold shift)
* automatically sets shiny input value - character vector with labels of selected nodes

Deployed example can be checked [here](https://connect.appsilon.com/content/f33a37e5-2e2e-4de8-8d16-f22dc330c422/_w_0b7d7274/#!/Tree).

### CODE REVIEW

**Review by averissimo — APPROVED** [review]
Looks good to me. Nice code cleanup.

My only comment is about the extensions of the imports. This is really optional, but I think it's good practice.

File: js/src/index.js [hunk 1]
```diff
@@ -6,10 +6,11 @@ import * as BlueprintSelect from '@blueprintjs/select';
...
 import * as Inputs from './inputs.jsx';
-import * as Suggests from './suggest.js';
-import * as Selects from './select.js';
-import * as MultiSelect from './multiselect.js';
-import * as MultiSlider from './multiSlider.js';
+import Suggest from './suggest.js';
+import Select from './select.js';
+import MultiSelect from './multiselect.js';
+import MultiSlider from './multiSlider.js';
+import Tree from './tree.js';
```
Comment (averissimo) [inline js/src/index.js:13]: I believe there's no need for the the extensions, the code below should work

```
import * as Inputs from './inputs';
import Suggest from './suggest';
import Select from './select';
import MultiSelect from './multiselect';
import MultiSlider from './multiSlider';
import Tree from './tree';
```

File: js/src/multiselect.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { MenuItem } from '@blueprintjs/core';
-import { MultiSelect2 } from '@blueprintjs/select';
-import { highlightText } from './utils/highlight-text.js';
+import React from "react";
+import PropTypes from "prop-types";
+import { MenuItem } from "@blueprintjs/core";
+import { MultiSelect2 } from "@blueprintjs/select";
+import { highlightText } from "./utils/highlight-text.js";
```
Comment (averissimo) [inline js/src/multiselect.js:5]: Same comment about extension

File: js/src/suggest.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { MenuItem } from '@blueprintjs/core';
-import { Suggest2 } from '@blueprintjs/select';
-import { highlightText } from './utils/highlight-text.js';
+import React from "react";
+import PropTypes from "prop-types";
+import { MenuItem } from "@blueprintjs/core";
+import { Suggest2 } from "@blueprintjs/select";
+import { highlightText } from "./utils/highlight-text.js";
```
Comment (averissimo) [inline js/src/suggest.js:5]: Same comment about extension

File: js/src/select.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { MenuItem, Button } from '@blueprintjs/core';
-import { Select2 } from '@blueprintjs/select';
-import { highlightText } from './utils/highlight-text.js';
+import React from "react";
+import PropTypes from "prop-types";
+import { MenuItem, Button } from "@blueprintjs/core";
+import { Select2 } from "@blueprintjs/select";
+import { highlightText } from "./utils/highlight-text.js";
```
Comment (averissimo) [inline js/src/select.js:5]: Same comment about extension

File: js/src/index.js (hunk 1 above)
Comment (pawelchabros) [reply js/src/index.js:13]: Extensions removed :+1:

File: js/src/multiselect.js (hunk 2 above)
Comment (pawelchabros) [reply js/src/multiselect.js]: Removed

File: js/src/suggest.js (hunk 3 above)
Comment (pawelchabros) [reply js/src/suggest.js]: Removed

File: js/src/select.js (hunk 4 above)
Comment (pawelchabros) [reply js/src/select.js]: Removed

**Pull request merged**